packages/libs/eda/src/lib/core/utils/variable-display.ts

```diff
@@ -1,9 +1,29 @@
 import { NumberVariable, VariableTreeNode } from '../types/study';
 
+//DKDK add scale in conjunction with
 export function variableDisplayWithUnit(
   variable: VariableTreeNode | undefined
 ): string | undefined {
-  if (NumberVariable.is(variable) && variable.units && variable.units != null)
-    return variable.displayName + ' (' + variable.units + ')';
-  else return variable?.displayName;
+  if (NumberVariable.is(variable)) {
+    if (variable.units && variable.units != null) {
+      if (variable.scale && variable.scale != null) {
+        return (
+          variable.displayName +
+          ' (' +
+          variable.units +
+          ', ' +
+          variable.scale +
+          ')'
+        );
+      } else {
+        return variable.displayName + ' (' + variable.units + ')';
+      }
+    } else {
+      if (variable.scale && variable.scale != null) {
+        return variable.displayName + ' (' + variable.scale + ')';
+      } else {
+        return variable?.displayName;
+      }
+    }
+  } else return variable?.displayName;
```

Comment:
There is quite a bit of nesting of if-statements here. I think we can reduce the this significantly

```suggestion
  if (variable == null) return;
  if (!NumberVariable.is(variable)) return variable.displayName;
  
  const parentheticalParts = [ variable.units, variable.scale ].filter(Boolean);
  
  if (parentheticalParts.length) return `${variable.displayName} (${parentheticalParts.join(', ')})`;

  return variable.displayName;
```